pf9/modules/analytics_utils.py

```diff
@@ -100,3 +113,12 @@ def send_identify(self, email, user_id):
         except Exception as except_err:
             logger.error("Exception in send_identify while communicating to segment")
             logger.exception(except_err)
+
+    def send_group(self, user_id, du_account_url):
+        try:
+            analytics.group(user_id, du_account_url, traits={'ddu_url_': 'DU', 'account_url_': du_account_url,
```

Comment:
What's the purpose for ddu_url_ trait?

Reply:
Its like a fixed trait pre-configured in segment based on which grouping is made.